README.md

```diff
@@ -11,7 +11,7 @@ Add `rspec-rails` to **both** the `:development` and `:test` groups in the
 
 ```ruby
 group :development, :test do
-  gem 'rspec-rails', '~> 3.0'
+  gem 'rspec-rails', '~> 3.4.2'
```

Comment:
We'd rather not have to update this all the time, if we were to merge this we'd want it to read `~> 3.4`